setup.py

```diff
@@ -16,7 +16,7 @@
 here = os.path.abspath(os.path.dirname(__file__))
 
 requirements = [
-        "bitarray>=0.8.1",
+        "bitarray-hardbyte>=0.8.0",
```

Comment:
Why did the patch number change?

Reply:
Just good practice for the `setup.py` file to be as broadly accepting of versions as possible - I don't think we actually require that particular patch so I removed that strict requirement.